packages/eslint-plugin-twenty/rules/no-use-hotkeys.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
```

Comment:
We need to export this rule in index.js

Reply:
No you need to export this rule along the others in `packages/eslint-plugin-twenty/index.js`

Reply:
You should use it into twenty/packages/eslint-plugin-twenty/index.js and remove rules key from module.exports.